providers/discovery/external/options.go

```diff
@@ -35,6 +36,13 @@ func Collective(c string) DiscoverOption {
 	}
 }
 
+// Federations returns the list of federated collectives to discover in
```

Comment:
...sets the federated collectives to discover in

Reply:
oops